nucleus/payara-modules/healthcheck-core/src/main/java/fish/payara/nucleus/healthcheck/admin/HistoricHealthCheckEventRetriever.java

```diff
@@ -110,11 +105,6 @@ public void execute(AdminCommandContext context) {
             actionReport.setActionExitCode(ActionReport.ExitCode.FAILURE);
             return;
         }      
-        if (!service.isHistoricalTraceEnabled()) {
-            actionReport.setMessage("Health Check Historical Trace is not enabled!");
-            actionReport.setActionExitCode(ActionReport.ExitCode.FAILURE);
```

Comment:
@MattGill98 this is being consumed from admin console and if we return failure, it throws out an error while opening the historic configuration page.